mobile/library/java/io/envoyproxy/envoymobile/engine/AndroidNetworkMonitor.java

```diff
@@ -63,7 +64,6 @@ static class DefaultNetworkCallback extends NetworkCallback {
         NetworkCapabilities.TRANSPORT_CELLULAR,  NetworkCapabilities.TRANSPORT_WIFI,
         NetworkCapabilities.TRANSPORT_BLUETOOTH, NetworkCapabilities.TRANSPORT_ETHERNET,
         NetworkCapabilities.TRANSPORT_VPN,       NetworkCapabilities.TRANSPORT_WIFI_AWARE,
-        NetworkCapabilities.TRANSPORT_LOWPAN,
```

Comment:
Why not including TRANSPORT_LOWPAN for SDK27 and above?

Reply:
But that makes me think that we should have some kind of CI check if we use an API that's not available in the minimum SDK version similar to what we have internally. I'm sure that'll be super useful. Let me think about it.

Reply:
Shouldn't it be something like below?
```
#ifdef SOME_SDK27_MACRO
TRANSPORT_TYPES = {..., TRANSPORT_WIFI_AWARE, TRANSPORT_LOWPAN};
#else
TRANSPORT_TYPES = {..., TRANSPORT_WIFI_AWARE};
#endif
```

Reply:
I asked mainly because I was unsure how TRANSPORT_LOWPAN would be handled in SDK27 and above with this change?

Reply:
Good question! Ideally we shouldn't want to hardcode the transport types at all but using an API like `getTransportTypes()`, but as far as I know, this API is considered a system API: https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/Connectivity/framework/src/android/net/NetworkCapabilities.java;l=1458-1467;drc=2fc115491a9c108707398aa56c1c3c26817f7178

These APIs seem to be hidden, too: https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/Connectivity/framework/src/android/net/NetworkCapabilities.java;l=1365-1377?q=TRANSPORT_WIFI_AWARE

In higher APIs, there are more transport types available and not just `LOWPAN`: https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/Connectivity/framework/src/android/net/NetworkCapabilities.java;l=1298-1302?q=TRANSPORT_WIFI_AWARE

Let me try to find a way to see if there's a way to programmatically get the list of transport types without hardcoding them.